mycroft/skills/__main__.py

```diff
@@ -199,9 +199,8 @@ def main(ready_hook=on_ready, error_hook=on_error, stopping_hook=on_stopping,
     bus = _start_message_bus_client()
     _register_intent_services(bus)
     event_scheduler = EventScheduler(bus)
-    status = ProcessStatus('skills', bus, on_started=ready_hook,
-                           on_error=error_hook,
-                           on_stopping=stopping_hook)
+    status = ProcessStatus('skills', bus, on_ready=ready_hook,
+                           on_error=error_hook, on_stopping=stopping_hook)
```

Comment:
This changes the behavior slightly, the ready_hook will be much later. Not sure if it'll be an issue but we should verify.

Reply:
At the highest level, this seemed odd to link the on_started handler with the ready_hook.

From my read, in the Skill Manager, `self._loaded_status` is set to True immediately after the `mycroft.skills.initialized` Message is emitted. This Message is what triggers the Padatious training. Once Padatious is finished the initial train it emits the `mycroft.skills.trained` Message. This Message is consumed by the enclosure which starts the `is_device_ready()` check.

So by this definition of "ready", I didn't think anything would change, but I expect that I'm mistaken...

Reply:
Meybe just add an on_alive / on started so the system can be informed at an earlier stage that loading of skills has worked (priority skills) and the process can be considered running since it's starting the main sequence of operation.